lldb/bindings/python/CMakeLists.txt

```diff
@@ -96,7 +96,9 @@ function(finish_swig_python swig_target lldb_python_bindings_dir lldb_python_tar
     ${lldb_python_target_dir}
     "utils"
     FILES "${LLDB_SOURCE_DIR}/examples/python/in_call_stack.py"
-          "${LLDB_SOURCE_DIR}/examples/python/symbolication.py")
+          "${LLDB_SOURCE_DIR}/examples/python/parsed_cmd.py"
```

Comment:
Sure.  I moved the `parsed_cmd.py` file into lldb/examples/python/templates as well, since the other files were there.  TTTT, that seems like a strange location for these files.  At this point they really aren't examples, they are base classes we're vending.  That's also true of things like crashlog, however, so I don't feel the need to address this right now.